cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/AppFabricServer.java

```diff
@@ -83,7 +83,7 @@ public class AppFabricServer extends AbstractIdleService {
   private final ProgramNotificationSubscriberService programNotificationSubscriberService;
   private final ProgramStopSubscriberService programStopSubscriberService;
   private final RunRecordCorrectorService runRecordCorrectorService;
-  private final RunRecordTimeToLiveService runRecordTimeToLiveService;
+  private final RecordTimeToLiveService recordTimeToLiveService;
```

Comment:
I don't really like new name. "Record" is too generic. Which record is that? If you wish to have a better name, I'd do something like "RunDataTimeToLiveService" since we are clearing only run-related things

Reply:
Done